posixlib/src/main/scala/scala/scalanative/posix/arpa/inet.scala

```diff
@@ -22,6 +22,19 @@ object inet {
   @name("scalanative_ntohs")
   def ntohs(arg: uint16_t): uint16_t = extern
 
+  /* The argument for inet_ntoa() differs from the POSIX specification
```

Comment:
Since `scaladoc` get generated, you can use `/** followed by */ and do the standard doc comments. On the `in` entry, that would be the place to say that it is pointer but the spec says reference.